W-18196015-storage-duke (#366)
mulesoft/docs-monitoring: modules/ROOT/pages/performance-and-impact.adoc

Logs::
In raw format, your logs can occupy as much space as you receive with your pricing package. Note that the storage space you receive with your pricing package is used to store logs and metrics data. Because metrics do not take up much storage capacity, you can use the majority of your storage for raw log data.

Searchable logs::
You receive additional storage that is 10% of the size of purchased storage for indexes for searchable logs.
+

> Anypoint Monitoring creates indexes on raw logs to enable you to efficiently and quickly perform log search queries. Anypoint Monitoring indexes the most recent logs for searching, up to 10% of the total purchased storage size. You can download logs to access older logs. 

Reviewer: Can we add: 
You can download logs to access older logs. --> You can download logs to access older logs using the Archive API

Author: @rohbharadwaj Do you actually _download_ them in order to access them with the API? Or is it that you just retrieve older logs by using the API? How is downloading involved?

Reviewer: We use the API to download the logs.

> Anypoint Monitoring creates indexes on raw logs to enable you to efficiently and quickly perform log search queries. Anypoint Monitoring indexes the most recent logs for searching, up to 10% of the total purchased storage size. You can download logs to access older logs. 

Author: ```suggestion
Anypoint Monitoring creates indexes on raw logs to enable you to efficiently and quickly perform log search queries. Anypoint Monitoring indexes the most recent logs for searching, up to 10% of the total purchased storage size. You can retrieve older logs with the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/Metric%20Data/[Anypoint Monitoring Archive API]. 
```

> +

Author: Here, only the part beginning "Total storage..." (below) is new.

> Anypoint Monitoring creates indexes on raw logs to enable you to efficiently and quickly perform log search queries. Anypoint Monitoring indexes the most recent logs for searching, up to 10% of the total purchased storage size. You can retrieve older logs with the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/Metric%20Data/[Anypoint Monitoring Archive API]. 

Author: @rohbharadwaj Reworded ☝️ to add Archive API. Can you check?

Logs::
In raw format, your logs can occupy as much space as you receive with your pricing package. Note that the storage space you receive with your pricing package is used to store logs and metrics data. Because metrics do not take up much storage capacity, you can use the majority of your storage for raw log data.

Searchable logs::

> <<<<<<< HEAD

Author: ```suggestion
```

Searchable logs::
You receive additional storage that is 10% of the size of purchased storage for indexes for searchable logs. +
Anypoint Monitoring creates indexes on raw logs to enable you to efficiently and quickly perform log search queries. Anypoint Monitoring indexes the most recent logs for searching, up to 10% of the total purchased storage size. You can download logs to access older logs. + 

> Total storage capacity is shared globally across all apps and is limited by the number of messages that your organization purchases and the maximum searchable storage allocation limit of 6TB.

Author: ```suggestion
Total storage capacity is shared globally across all apps and is limited by the maximum searchable storage allocation of 6TB. 
```

== Screen Resolution

Use a minimum screen resolution of 1024 by 768 pixels.

> == Grace Period for Data Retention

Author: ```suggestion
== Grace Period for Metrics Data Retention
```

> == Grace Period for Data Retention Prior to Decommissioning 

Author: ```suggestion
== Grace Period for Data Retention
```

If your organization downgrades to a lower subscription tier (usage-based or core-based) or cancels your subscription, your data is removed according to this schedule:

> 

Author: Rajesh will confirm periods

Author: he says 21 days is the grace period

== Screen Resolution

Use a minimum screen resolution of 1024 by 768 pixels.

== Grace Period for Metrics Data Retention

If your organization downgrades to a lower subscription tier (usage-based or core-based) or cancels your subscription, your data is removed according to this schedule:

* Downgrades: Approximately 14 days after a downgrade to a lower tier, an additional 21-day grace period takes place before your data is removed.

> * Cancellations: Approximately 90 days after a cancellation, an additional 21-day grace period takes place before your data is removed.

Reviewer: The 21 day grace period as per my knowledge is for the metrics storage and not the logs storage. Should we mention this here? or is there a nice way to mention this applied only to metrics. 
Need confirmation from Rajesh

Reviewer: The heading mentions Metrics. So we are good here.
